README.md

```diff
@@ -204,6 +205,64 @@ In order to enable searchable encryption, you will need to configure your Cipher
 - If you are using [CipherStash Proxy](https://github.com/cipherstash/proxy), see [this guide](docs/tutorials/proxy-configuration.md).
 - If you are using [Protect.js](https://github.com/cipherstash/protectjs), use the [Protect.js schema](https://github.com/cipherstash/protectjs/blob/main/docs/reference/schema.md).
 
+## Documentation
+
+### API Documentation
+
+All EQL functions and types are fully documented with Doxygen-style comments in the source code.
+
+**Install Doxygen** (required for documentation generation):
```

Comment:
Ideally we do this as a mise tool, but as none exists right now, this will have to do.